src/components/Explore/TokenTable/TokenRow.tsx

```diff
@@ -247,13 +244,13 @@ const TokenInfoCell = styled(Cell)`
 const TokenName = styled.div`
   overflow: hidden;
   text-overflow: ellipsis;
-  max-width: 160px;
+  max-width: 280px;
```

Comment:
<img width="550" alt="Screen Shot 2022-08-10 at 3 28 17 PM" src="https://user-images.githubusercontent.com/62825936/184003560-43316a11-2ca1-42b2-9601-3c6afcba6cc9.png">

Reply:
@lynnshaoyu i think this change overflows it on bigger screens, maybe make it this max when screen size is SMALL_MEDIA?

Reply:
<img width="441" alt="Screen Shot 2022-08-10 at 3 29 30 PM" src="https://user-images.githubusercontent.com/62825936/184003733-d782231d-506a-45bc-b914-22c1f916c86f.png">

Reply:
same here

Reply:
oof what width was this?

Reply:
as an fyi: we should generally make sure our smallest size layout works and doesn't break anything down to 375px (iphone 7/8 size)

Reply:
<img width="690" alt="Screen Shot 2022-08-10 at 3 31 35 PM" src="https://user-images.githubusercontent.com/41491154/184014459-8360b4ec-fcb9-4532-8e6a-353e738ba494.png">
this is the newest PR's render

Reply:
this is 375px
<img width="537" alt="Screen Shot 2022-08-10 at 3 49 27 PM" src="https://user-images.githubusercontent.com/41491154/184017359-691e45c2-ddde-499c-8b40-8b2fe07def1a.png">